Makefile

```diff
@@ -36,7 +36,7 @@ teams-clean: prepare
 	  rm -f $(TEAMS_DIR)/$${team}.html ;\
 	done
 
-teams: | teams-clean $(patsubst %,$(TEAMS_DIR)/%.md,$(TEAMS))
+teams: | teams-clean $(patsubst %,$(TEAMS_DIR)/%.md,$(TEAMS)) ##generates numpy.org team gallery pages
```

Comment:
Add space to be consistent.  That is
```
## generates numpy.org team gallery pages
```
instead of
```
##generates numpy.org team gallery pages
```